client/settings/payment-settings/general-settings-section.js

```diff
@@ -21,76 +20,48 @@ const GeneralSettingsSection = () => {
 	const [ enableStripe, setEnableStripe ] = useState( false );
 	const [ enableTestMode, setEnableTestMode ] = useState( false );
 
-	const toggleEnableStripe = () => setEnableStripe( ! enableStripe );
-	const toggleEnableTestMode = () => setEnableTestMode( ! enableTestMode );
-
-	const CheckboxDescription = styled.span`
-		font-style: italic;
-		display: block;
-	`;
-
-	const GeneralSettingsOptionWrapper = styled.div`
-		&:first-of-type {
-			margin-top: 26px;
-		}
-		&:not( :last-child ) {
-			margin-bottom: 26px;
-		}
-		&:last-child {
-			margin-bottom: 24px;
-		}
-	`;
-
 	return (
 		<Card>
 			<CardBody>
-				<GeneralSettingsOptionWrapper>
-					<CheckboxControl
-						checked={ enableStripe }
-						onChange={ toggleEnableStripe }
-						label={ __(
-							'Enable Stripe',
-							'woocommerce-gateway-stripe'
-						) }
-					/>
-					<CheckboxDescription>
-						{ __(
-							'When enabled, payment methods powered by Stripe will appear on checkout.',
-							'woocommerce-gateway-stripe'
-						) }
-					</CheckboxDescription>
-				</GeneralSettingsOptionWrapper>
+				<CheckboxControl
+					checked={ enableStripe }
+					onChange={ setEnableStripe }
+					label={ __(
+						'Enable Stripe',
+						'woocommerce-gateway-stripe'
+					) }
+					help={ __(
```

Comment:
On other sections we just use the `help` prop on the `*Control` component - it is not italic, but that's due to an error on the Figma components. We did the same in WCPay.
In Gutenberg, the `help` text is not italic as well: https://wordpress.github.io/gutenberg/?path=/story/components-basecontrol--default

Reply:
I noticed that in WCPay. The reason for the help text was to address the design. If both ways are doing the same thing, how do we decide to use `help` prop? (One way or the other?)

Reply:
That's totally fair!

I think the important thing is to ensure that we're using consistent spacing at least within the plugin, otherwise that's going to be subject to feedback during the UX review.

In this specific case, `*Control` components use the `help` prop to leverage the `aria-describedby` attribute for screen readers to provide additional information about an input. If we didn't use the `help` prop, we would need to find a way to use the `aria-describedby` attribute ourselves, which would be a bit harder
![Markup 2021-09-13 at 15 50 52](https://user-images.githubusercontent.com/273592/133154567-f86273fd-7346-4053-9b9d-0cd04ba35e18.png)

Reply:
👍  Ok thanks, sounds like the `help` props should be preferred whenever possible, due to 
- a11y
- 1 less component since it's built in
- safe to not match design 100% (italic/underline/padding etc)

Reply:
I was looking if Figma was providing a prop/name match on the components to help us infer which props we should be using on a component (i.e.: I was trying to see if the text in Figma was named "help" or something similar).
But I wasn't able to find an obvious match :( the components in Figma are _mostly_ called "Frame*", I couldn't find a way to infer the type of the component being used and its props

Reply:
Figma uses the same React component we are building? :O

Reply:
Figma uses Gutenberg components as "building blocks" - I was trying to find a match with the Gutenberg components